crds/crd-coralogix-alert-set.yaml

```diff
@@ -78,6 +78,54 @@ spec:
                                 descprition: Sends resolve notification when the alert criteria that you defined is no longer met
                                 type: boolean
                                 default: true
+                              metricAlertParameters:
+                                description: Data for defining metric
+                                type: object
+                                properties:
+                                  metricField:
+                                    type: string
+                                  metricSource:
+                                    type: string
+                                    enum:
+                                      - LOGS2METRICS
+                                      - PROMETHEUS
+                                  arithmeticOperator:
+                                    type: string
+                                    enum:
+                                      - AVG
+                                      - MIN
+                                      - MAX
+                                      - SUM
+                                      - COUNT
+                                      - PERCENTILE
```

Comment:
Saw in the UI that when you choose `PRECENTILE` it request also the value, we need to check if it supported